src/clue/adafruit_clue.py

```diff
@@ -199,15 +199,65 @@ class Clue:  # pylint: disable=too-many-instance-attributes, too-many-public-met
     RAINBOW = (RED, ORANGE, YELLOW, GREEN, BLUE, PURPLE)
 
     def __init__(self):
+        self._a = False
+        self._b = False
+        self.__pressed_buttons = set()
         self._pixel = neopixel.NeoPixel(
             pin=CONSTANTS.CLUE_PIN, n=1, pixel_order=neopixel.RGB
         )
 
+    @property
+    def button_a(self):
+        """``True`` when Button A is pressed. ``False`` if not.
+        This example prints when button A is pressed.
+        To use with the CLUE:
+        .. code-block:: python
+          from adafruit_clue import clue
+          while True:
+              if clue.button_a:
+                  print("Button A pressed")
+        """
+        return self._a
+
+    @property
+    def button_b(self):
+        """``True`` when Button B is pressed. ``False`` if not.
+        This example prints when button B is pressed.
+        To use with the CLUE:
+        .. code-block:: python
+          from adafruit_clue import clue
+          while True:
+              if clue.button_b:
+                  print("Button B pressed")
+        """
+        return self._b
+
+    def __update_button(self, button, value):
+        if button == "button_a":
+            if value:
+                self.__pressed_buttons.add("A")

```

Comment:
With this, it might be better to have a dict with each button mapped to whether they are pressed. This way, it might be a little clearer. 
Something like:
`pressed_btns["A"] = value or pressed_btns["A"]`

Reply:
Same as below, I implemented it so that it works the same as the CLUE library.
Here is the original:
```
        ret = set()
        pressed = self._gamepad.get_pressed()
        for button, mask in (("A", 0x01), ("B", 0x02)):
            if mask & pressed:
                ret.add(button)
        return ret
```